scripts/data_migrations/2019-04-25_update-british-population-topic.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE topic
+SET additional_description = 'In the 2011 Census, 80.5% of people in England and Wales said they were White British, and 19.5% were from ethnic minorities. Everyone completing the Census was asked to choose from a [list of ethnic groups](/ethnic-groups).'
```

Comment:
Hadn't really clocked this until now - but this feels like the first time we're including internal links within markdown? Feels like we could easily forget to update these things if we ever change routes in the future. Have we thought about/mitigated this at all?

Reply:
I'm not sure how it can be mitigated other than by properly redirecting.  We might have a bunch of measures where people have linked to internal pages in the markdown and we'd have no visibility of that at all.

Reply:
@samuelhwilliams I don't think it's strictly the first time, as I think some of the measures also link to each other via "related links".

I did briefly consider how best to handle these, but it seems most systems (including Whitehall publisher) don't do anything clever – so it's just up to us to make sure we set up redirects whenever change the URLs... (which is good practice anyway).

Reply:
So this PR is just because we've noticed it, not because we want to update the URLs as standard (because we rely on redirects being in place and working)?